### PR TITLE
rqg: Add the banking workload to the CI

### PR DESCRIPTION
**Pull request by philip-stoev**
Misc fixes to the rqg mzcompose and adding a new workload.

### Motivation

The RQG allows some concurrency + correctness testing so I wanted to get this into Buildkite in order to confirm that it runs green for the remainder of the month.

### Tips for reviewer

This PR can be reviewed commit by commit.

### CODE REVIEW

**Review by def- — APPROVED** [review]
Nice!

File: test/rqg/mzcompose.py [hunk 1]
```diff
@@ -27,29 +27,40 @@ class Dataset(Enum):
     SIMPLE = 1
     DBT3 = 2
 
+    def files(self) -> list[str]:
+        match self:
+            case Dataset.SIMPLE:
+                return ["simple.sql"]
+            case Dataset.DBT3:
+                return ["dbt3-ddl.sql", "dbt3-s0.0001.dump"]
```
Comment (nrainer-materialize) [inline test/rqg/mzcompose.py:35]: These files are not part of this PR. Where are they located? Where they already introduced with another PR or ar they outside of the repo?

**Review by nrainer-materialize — APPROVED** [review]

File: test/rqg/mzcompose.py (hunk 1 above)
Comment (philip-stoev) [reply test/rqg/mzcompose.py:35]: For license purity reasons, they are in the MaterializeInc/RQG repository.
